CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.2.0] - TBD 
```

Comment:
Make sure to update this in the `master` branch after the collapse before we publish this to supermarket.